Vagrantfile

```diff
@@ -0,0 +1,43 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+@script = <<SCRIPT
+DOCUMENT_ROOT_ZEND="/var/www/zf/public"
```

Comment:
I see the sync'd folder done as `/vagrant` quite often. What do people think about syncing `/var/www/zf` Like this? I'm not sure how I feel about it.